### PR TITLE
Blog - Working With Zip Files in Go

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
Preview URL:
https://64187ca2df97a65dc82dfe59--ephemeral-narwhal-3ac3b2.netlify.app

@jalletto

**Comment by balapriyac** [comment]
https://64187ca2df97a65dc82dfe59--ephemeral-narwhal-3ac3b2.netlify.app/blog/golang-zip-files/

File: blog/_posts/2023-04-30-golang-zip-files.md [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+go run main.go
+~~~
+
+<div class="wide">
+![add to zip file]({{site.images}}{{page.slug}}/nOtKySW.png)
+</div>
+
+If you try to run the code and `newfile.txt` does not exist, you will get the following output:
+
+~~~{.bash caption="Output"}
```
Comment (balapriyac) [inline blog/_posts/2023-04-30-golang-zip-files.md:361]:
```suggestion
~~~{ caption="Output"}
```

File: blog/_posts/2023-04-30-golang-zip-files.md [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+            panic(err)
+        }
+        reader.Close()
+    }
+}
+}
+
+~~~
+
+Before running the code, create a new file named `*newfile.txt*` in the same directory as `*main.go*`. To check the output go to the terminal and run the following:
```
Comment (balapriyac) [inline blog/_posts/2023-04-30-golang-zip-files.md:349]:
```suggestion
Before running the code, create a new file named `newfile.txt` in the same directory as `main3.go`. To check the output go to the terminal and run the following:
```

**Review by balapriyac — CHANGES_REQUESTED** [review]
Thanks, Sameer.

**Comment by msali123** [comment]
Thanks Bala, all changes done.

https://641c18256b738b10329ad233--ephemeral-narwhal-3ac3b2.netlify.app/blog/golang-zip-files/

**Review by jalletto — APPROVED** [review]

**Pull request merged**